lib/src/core/selection.dart

```diff
@@ -95,7 +95,7 @@ class SelectionController<T> extends Listenable
 
   /// Returns true if controller was never in the in selection state.
   bool get wasEverSelected => _wasEverSelected;
-  bool _wasEverSelected = false;
+  final bool _wasEverSelected = false;
```

Comment:
This is never updated, hence the note about that it could be `final`. Is the fact that this is never set a bug?